diffkemp/simpll/simpll_lib.py

```diff
@@ -1,15 +1,11 @@
-import sys
-from diffkemp.utils import get_simpll_build_dir
-
 # Dynamically import the SimpLL C extension module.
 # First try to import from the current build directory (to allow multiple
 # parallel local builds).
 # If that fails, import from the default location.
 try:
-    sys.path.append(get_simpll_build_dir())
-    _simpll = __import__("_simpll")
+    _simpll = __import__('_simpll')
 except ImportError:
-    _simpll = __import__("diffkemp.simpll._simpll", fromlist=[None])
+    _simpll = __import__('diffkemp.simpll._simpll')
```

Comment:
This could be probably removed (it is not probably used) because of the change in the `simpll_build.py` file of `diffkemp.simpll._simpll` to just `_simpll` (the `_simpll` 'library/package' is not saved in the `diffkemp/simpll` subfolder anymore but it is saved in diffkemp/`site_packages` directory instead).

Reply:
I think that it is used, but I have to do a short experiment to be really sure, it is saved in `site-packages` but it still under the path `somewhere-in-site-packages/diffkemp/diffkemp/simpll/libsimpll.so`. But again, my confidence in this is not high enough, so I am going to do a quick experiment.

You have also uncovered some code style error from my side, I am used to using single quotes but this project is using double quotes, I am going to fix that, thanks.